src/core/ipfs/go-online.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const series = require('run-series')
+const Bitswap = require('ipfs-bitswap')
+
+module.exports = function goOnline (self) {
+  return (cb) => {
+    series([
+      self.load,
+      self.libp2p.start
+    ], (err) => {
+      if (err) {
+        return cb(err)
+      }
+
+      self._bitswap = new Bitswap(
+        self._peerInfo,
+        self._libp2pNode,
+        self._repo.datastore,
+        self._peerInfoBook
+      )
+      self._bitswap.start()
+      self._blockS.goOnline(self._bitswap)
```

Comment:
I'm a bit confused why doesn't blockService start bitswap, if in order for blockService to go online, it requires bitswap to go online